lib/TransactionRunner.js

```diff
@@ -350,6 +351,11 @@ class TransactionRunner {
     } else {
       results = transaction.test.results = transaction.results;
     }
+
+    // Ensure transaction errors are propagates to the test.
+    // That way transaction errors are a part of an Apiary Reporter.
+    transaction.test.errors = transaction.errors;
```

Comment:
This is needed in such utils because the `transaction.test` instance may already be present, and at the moment of skipping/failing contain no errors per this line:

```js
test.errors = transaction.errors
```

As the skip/failure happens _after_ the test run, we need to ensure the `errors` are set explicitly.